Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'pmp', '~> 0.5'
 
 gem 'hyperresource'
 
+gem 'addressable'
```

Comment:
FWIW I built the request path as a string to include `fragment`, idk if that would ever actually matter